homeassistant/components/telegram_bot/__init__.py

```diff
@@ -439,6 +459,16 @@ def _render_template_attr(data, attribute):
             await hass.async_add_executor_job(
                 partial(notify_service.delete_message, **kwargs)
             )
+        elif msgtype in [
```

Comment:
`msgtype` is not a good variable name anymore, rename it to `service` or something like that